packages/ui-extensions-server-kit/src/ExtensionServerClient/ExtensionServerClient.test.ts

```diff
@@ -10,12 +10,15 @@ const defaultOptions = {
 describe('ExtensionServerClient', () => {
   let socket: WS
 
-  function setup(options: ExtensionServer.Options = defaultOptions) {
+  async function setup(options: ExtensionServer.Options = defaultOptions) {
```

Comment:
Related to hydrogen? or some other fix?

Reply:
seems unrelated to the changes in this PR, not sure why is failing now.